futurenhs.app/components/Reply/index.tsx

```diff
@@ -21,9 +21,11 @@ export const Reply: (props: Props) => JSX.Element = ({
 }) => {
     const wrapperRef = useRef()
 
-    const formConfig: FormConfig = selectForm(
-        forms,
-        formTypes.CREATE_DISCUSSION_COMMENT_REPLY
+    const replyForm: FormConfig = forms[formTypes.CREATE_DISCUSSION_COMMENT_REPLY]
```

Comment:
hmm I think they are both the same but the config from the page held the initial values. It was mainly just to avoid duplication/increase readability when passing into the hook instead of the below for example. Though the other suggested change I've made has removed the need for this

`useFormConfig(formTypes.CREATE_GROUP, forms[formTypes.CREATE_GROUP].initialValues, forms[formTypes.CREATE_GROUP].errors`